profiles/templates/profiles/profile.html

```diff
@@ -6,4 +6,23 @@
     <!-- TO DO : build profile page -->
     TEST PROFILE PAGE
 
+<!DOCTYPE html>
+    <h1>Profile Page</h1>
+    <h2>You are logged in as</h2>
+
+<!---------------------------------------profile display------------------------------------------------->    
+    <div class="container-fluid">
+      <div class="row">
+        <div class="col-md-4"><h4>Username :</h4><p> {{ user.username }} </p></div>

```

Comment:
@aticodein 
Since these lines will eventually grow to more content or design/styles, let's try to break them up with some code formatting, instead of using everything on a single-line. Something like this, for example:

```
<div class="col-md-4">
    <h4>Username:</h4>
    <p>{{ user.username }}</p>
</div>
```

The same would apply to the other lines below.
Also, watch out for extra spaces where they're not needed.

Reply:
Changed to expected formating.